pkgs/development/php-packages/msgpack/default.nix

```diff
@@ -0,0 +1,18 @@
+{ buildPecl, lib }:
+
+let
+  version = "2.2.0RC2";
```

Comment:
I don't think it's needed to have `version` in a `let` ... `in` statements.

Reply:
I need it for meta.changelog

Reply:
You could use `buildPecl rec {` then ?

Reply:
alright